capstone/src/main/webapp/script.js

```diff
@@ -334,18 +335,22 @@ const requestParamPOST = {
   },
 };
 
+/** Obtains Reviews -- hardcoded examples for now */
 function getReviews() {
   // TODO(#33): integrate with actual reviews of businesses
-  const review = 'Really good pizza, nice wine, reasonable prices and great music.';
-  getEntities(review);
+  const review =
```

Comment:
Just to double check, this is a linter change right?

Reply:
Yes, that was a linter change